### PR TITLE
improve page title and main navigation for screenreaders

### PR DESCRIPTION
**Pull request by sknep**
Now, screenreaders will announce the page title when they change routes. Also this removes the "null" present in page titles for our example libraries and uses the temporary names we've given them on the front end.

### CODE REVIEW

**Comment by sknep** [comment]
Please don't merge without Robert's approval, thanks!

**Review by iamjolly — APPROVED** [review]
This looks like what we talked about. I am not running it locally, so let's approve this and I can test it out on the server with a screen reader to hopefully confirm the desired behavior.

**Pull request merged**
